pkg/controllers/scheduling/schedule.go

```diff
@@ -140,6 +142,12 @@ func (s *pluginScheduler) Schedule(
 	for _, cluster := range filtered {
 		scoreSum[cluster.Name] = 0
 	}
+	// define steady prioritizer weight, default is PolicyTypeNotSteady
+	steadyWeight := 0.1
+	if placement.Spec.ChurningPolicy.PolicyType == clusterapiv1alpha1.PolicyTypeSteady {
+		steadyWeight = float64(len(s.prioritizers) + 1)
```

Comment:
I think you would want a map[string]int for weight and give it init values. You could have a separate func to adjust weight.

Reply:
As discussed, remove steady related code.